### PR TITLE
[Ruby/pt-rb] Add modules explanation as in main file

### PR DESCRIPTION
**Pull request by DemonKart**
Add the modules explanation (translated) to the brasilian portuguese file.
---
Adiciona explicação de módulos (traduzida) ao arquivo português brasileiro.

Signed-off-by: demonkart <alanpcsilva@gmail.com>

### CODE REVIEW

**Comment by vendethiel** [comment]
Sorry, it seems there's a merge conflict. I can merge it if you fix it (also please move yourself as a translator)

**Comment by DemonKart** [comment]
Ok, I will solve the conflict ASAP.

**Comment by vendethiel** [comment]
Thanks!

**Comment by DemonKart** [comment]
Just solved the conflict and moved me as a translator as asked :)

File: pt-br/ruby-pt.html.markdown [hunk 1]
```diff
@@ -382,12 +383,75 @@ end
...
+  include ExemploDeConceito
+end
+
+Algo.bar     # => 'bar'
+Algo.qux     # => NoMethodError: undefined method `qux'
+Algo.new.bar # => NoMethodError: undefined method `bar'
+Algo.new.qux # => 'qux'
 ```
 
+<<<<<<< HEAD
```
Comment (vendethiel) [inline pt-br/ruby-pt.html.markdown]: Whoops, looks like there's some merge artifact left

**Comment by vendethiel** [comment]
Could you remove those marks?

**Comment by DemonKart** [comment]
Sorry about taking this long to just remove this :s I totally forgot about it, but now it is ok.

File: pt-br/ruby-pt.html.markdown [hunk 2]
```diff
@@ -382,12 +383,74 @@ end
 Humano.bar # 0
 Doutor.bar # nil
 
+---
```
Comment (vendethiel) [inline pt-br/ruby-pt.html.markdown]: still some marks

File: pt-br/ruby-pt.html.markdown [hunk 3]
```diff
@@ -382,12 +383,74 @@ end
...
+- [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) - Um guia mantido pela comunidade de estilo de código Ruby.
+=======
 ## Mais sobre Ruby
 
 - [Documentação oficial](http://www.ruby-doc.org/core-2.1.1/)
 - [Aprenda Ruby com desafios](http://www.learneroo.com/modules/61/nodes/338) - Uma coleção de desafios para testar a linguagem.
 - [Ruby a partir de outras linguagens](https://www.ruby-lang.org/en/documentation/ruby-from-other-languages/)
 - [Programming Ruby](http://www.amazon.com/Programming-Ruby-1-9-2-0-Programmers/dp/1937785491/)- Um mais antigo [free edition](http://ruby-doc.com/docs/ProgrammingRuby/) e tambem uma versão online disponível.
 - [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) - Uma versão colaborativa de um *style-guide*
+>>>>>>> df4dbee36f5a3b9b63292a2346327f8b7d282a28
```
Comment (vendethiel) [inline pt-br/ruby-pt.html.markdown]: here

File: pt-br/ruby-pt.html.markdown [hunk 4]
```diff
@@ -382,12 +383,74 @@ end
...
+
+Algo.bar     # => 'bar'
+Algo.qux     # => NoMethodError: undefined method `qux'
+Algo.new.bar # => NoMethodError: undefined method `bar'
+Algo.new.qux # => 'qux'
 ```
 
+## Recursos adicionais
+
+- [Learn Ruby by Example with Challenges](http://www.learneroo.com/modules/61/nodes/338) - A variant of this reference with in-browser challenges.
```
Comment (vendethiel) [inline pt-br/ruby-pt.html.markdown]: please mark english links with a " (EN)"  tag

File: pt-br/ruby-pt.html.markdown [hunk 5]
```diff
@@ -6,6 +6,7 @@ contributors:
   - ["Bruno Henrique - Garu", "http://garulab.com"]
   - ["Jean Matheus Souto", "http://jeanmatheussouto.github.io"]
 translators:
+  - ["Alan Peterson Carvalho Silva", "https://twitter.com/DemonKart"]
```
Comment (vendethiel) [inline pt-br/ruby-pt.html.markdown]: if you're gonna add yourself as a translator, please don't add yourself as the first one

**Comment by DemonKart** [comment]
done

**Comment by vendethiel** [comment]
Looks good, thanks!

**Pull request merged**